examples/virtualappliance/package.json

```diff
@@ -1,16 +1,14 @@
 {
-    "name": "aws-typescript",
+    "name": "virtualappliance",
     "devDependencies": {
         "@types/node": "latest"
     },
     "dependencies": {
-        "@pulumi/aws": "2.0.0-beta.1",
-        "@pulumi/pulumi": "2.0.0-beta.3",
-        "@pulumi/random": "2.0.0-beta.2",
+        "@pulumi/aws": "^2.0.0",
+        "@pulumi/pulumi": "^2.0.0",
+        "@pulumi/random": "^2.0.0",
         "@types/node-fetch": "^2.1.4",
-        "node-fetch": "^2.3.0"
-    },
-    "peerDependencies": {
-        "@pulumi/f5bigip": "dev"
+        "node-fetch": "^2.3.0",
+        "@pulumi/f5bigip": "^2.0.0"
```

Comment:
Why remove `peerDependencies`? I think we use them elsewhere (e.g. https://github.com/pulumi/pulumi-aws/blob/master/examples/minimal/package.json#L18)

Reply:
same comment as before - we are moving away from these - I will be sending a PR to the AWS repo to kill off peerDependencies so we can get some upgrade tests going